### ISSUE TITLE
Update all out-of-date modules

### ISSUE THREAD

**Issue by dankohn** [cncf/landscape#926]
Using `yarn outdated`.

See if it resolves:
```
1:49:27 PM: (node:1399) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
1:49:33 PM: (node:1399) DeprecationWarning: Chunk.mapModules: Use Array.from(chunk.modulesIterable, fn) instead
```

**Comment by ZeusTheTrueGod** [comment]
We can not update `jimp`, because its changes are incompatible with our code.
I'll check if we can upgrade all other modules

**Comment by dankohn** [comment]
Closed with 9a31fe01ae21cc8d9753d08629119d6ac791babd
